src/lib/services/DependencyChecker.ts

```diff
@@ -0,0 +1,11 @@
+
+
+export interface DependencyChecker {
+	getName(): string;
+
+	run(target?: string): Promise<[boolean, string, string]>;
```

Comment:
It would be clearer if this returned a strongly typed object instead of an array.

Reply:
Agreed. It's definitely not going to stay that way. I've just got it this way for now because it's what was easiest.